package.json

```
@@ -26,7 +26,7 @@
   "typings": "index.d.ts",
   "homepage": "https://github.com/auth0/angular2-jwt",
   "peerDependencies": {
-    "@angular/common": "^6.0.0"
+    "@angular/common": ">=6.0.0"
```

Suggest changing to:
```suggestion
    "@angular/common": ">= 6.0.0 < 8.0.0"
```

This way it will not accidentally break if Angular 8 releases and does have a breaking change with this library.

---

I switched it to just `>=6.0.0` as it doesn't look like the authors of this repo have that much time to dedicate to it, rather than having to update the max version every 6 months with a new angular release, it seemed better to just have a minimum version

---

@mattlewis92 fair enough. Probably better to simply fix it later if it ends up becoming a problem.